frontend/src/utils/MapMarkers.tsx

```diff
@@ -31,14 +31,14 @@ export const PlaceTagsInMap = (mission: Mission, map: HTMLCanvasElement, current
     })
 }
 
-export const PlaceRobotInMap = (mapMetadata: MapMetadata, map: HTMLCanvasElement, robotPose: Pose) => {
+export const placeRobotInMap = (mapMetadata: MapMetadata, map: HTMLCanvasElement, robotPose: Pose) => {
     const pixelPosition = calculateObjectPixelPosition(mapMetadata, robotPose.position)
     const rad = calculateNavigatorAngle(robotPose)
     drawRobotMarker(pixelPosition[0], pixelPosition[1], map, 22)
     drawNavigator(pixelPosition[0], pixelPosition[1], map, rad)
 }
 
-export const InverseCalculatePixelPosition = (mapMetadata: MapMetadata, pixelPosition: ObjectPosition) => {
+export const inverseCalculatePixelPosition = (mapMetadata: MapMetadata, pixelPosition: ObjectPosition) => {
```

Comment:
I don't fully understand this name? Is it inverting it or is an inverse calculation something else? We could maybe call it getInversePixelPosition /calculateInversePixelPosition or something similar to make it easier to understand?

Reply:
There already is a calculatePixelPosition function and this gives the inverse of that function